src/github.com/getlantern/flashlight/app/pac.go

```diff
@@ -24,7 +24,7 @@ import (
 var (
 	isPacOn       = int32(0)
 	pacURL        string
-	pacURLNoCache string
+	pacURLNoCache atomic.Value
```

Comment:
@myleshorton @xiam While testing the masquerade caching, the race detector uncovered this race condition.  I went ahead and just fixed it in this branch.